### PR TITLE
Add log shipper configuration docs

### PR DESCRIPTION
**Pull request by raskchanky**

### CODE REVIEW

**Comment by mladlow** [comment]
Is there new telemetry that needs to go on the telemetry page? You could try deleting your node_modules folder in the website dir, that helps sometimes.

**Comment by raskchanky** [comment]
@mladlow Thanks for the reminder about the telemetry. I'll add that to this. I did finally sort out the node errors.

File: website/content/docs/configuration/replication.mdx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+replication {
+  resolver_discover_servers = true
+  logshipper_buffer_length = 1000
+  logshipper_buffer_size = "5gb"
+}
+```
+
+## `replication` Parameters
+
+- `resolver_discover_servers` `(boolean: true)` - Indicates if the resolver should discover servers.
```
Comment (ncabatoff) [inline website/content/docs/configuration/replication.mdx:23]: It's probably fine to leave this as-is, but I could see it leading to people opening issues to ask for more details.  To pre-empt that, we could explain:

This feature should probably always be turned on.  It enables secondary cluster nodes to reach out to nodes in the primary cluster to request information on who the active node is. This fixes an issue where a secondary cluster may only know about a single node in the primary cluster, and if that node isn't the active node the replication connection will not succeed. With resolving disabled, the only way around this is to go through an onerous UpdatePrimary workflow, or do a step-downs on the primary cluster until the one node known by the secondary cluster is active.

File: website/content/docs/configuration/replication.mdx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+- `logshipper_buffer_length` `(integer: 0)` - The maximum number of entries that the log shipper buffer can hold.
+- `logshipper_buffer_size` `(string: "")` - The maximum size that the log shipper buffer can be. This size can be
```
Comment (ncabatoff) [inline website/content/docs/configuration/replication.mdx:25]: Should we add more details about what happens when the value isn't set?

File: website/content/docs/configuration/replication.mdx (hunk 1 above)
Comment (raskchanky) [reply website/content/docs/configuration/replication.mdx:23]: Thank you for this. I wasn't entirely sure what to put here. 😅

File: website/content/docs/configuration/replication.mdx (hunk 2 above)
Comment (raskchanky) [reply website/content/docs/configuration/replication.mdx:25]: Yeah, good call.

**Comment by raskchanky** [comment]
@ncabatoff Updated with your suggestions

**Review by ncabatoff — APPROVED** [review]

File: website/content/docs/configuration/replication.mdx [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+# `replication` Stanza
+
+The `replication` stanza specifies various parameters for tuning replication related values.
+
+```hcl
+replication {
+  resolver_discover_servers = true
+  logshipper_buffer_length = 1000
+  logshipper_buffer_size = "5gb"
+	allow_forwarding_via_header = false
```
Comment (ncabatoff) [inline website/content/docs/configuration/replication.mdx:18]:
```suggestion
 	allow_forwarding_via_header = false
```

File: website/content/docs/configuration/replication.mdx [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+	best_effort_wal_wait_duration = "2s"
```
Comment (ncabatoff) [inline website/content/docs/configuration/replication.mdx:19]:
```suggestion
	 best_effort_wal_wait_duration = "2s"
```

**Pull request merged**